knpu/episode3/Challenges/AddingAbstract/Coding.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Challenges\AddingAbstract;
+
+use KnpU\ActivityRunner\Activity\CodingChallenge\CodingContext;
+use KnpU\ActivityRunner\Activity\CodingChallenge\CorrectAnswer;
+use KnpU\ActivityRunner\Activity\CodingChallengeInterface;
+use KnpU\ActivityRunner\Activity\CodingChallenge\CodingExecutionResult;
+use KnpU\ActivityRunner\Activity\CodingChallenge\FileBuilder;
+use KnpU\ActivityRunner\Activity\Exception\GradingException;
+
+class Coding implements CodingChallengeInterface
+{
+    /**
+     * @return string
+     */
+    public function getQuestion()
+    {
+        return <<<EOF
+When Darth is browing all the different `DeathStar` models,
+we want to print out a little description that describes
+each one. Inside of that description, we want to include
+the "laser range", but it's different based on the model.
+Use an abstract method to return descriptions with the
```

Comment:
@weaverryan I don't sure that I understand it correctly... An `abstract` method can not contain a body, so I should move method's body to the DeathStar and DeathStarII classes and make this method as abstract in AbstractDeathStar?